heltour/tournament/views.py

```diff
@@ -1397,6 +1450,9 @@ def team(season):
             'season_perf_rating': season_perf_rating,
             'season_score': season_score,
             'season_score_total': season_score_total,
+            'career_performance': performances[0],
```

Comment:
Again, prefer deconstruction (or just use separate variables).
If you want compactness, you can assign multiple related variables on the same line. For example,
`a, b, c = d, e, f`.